roles/lnls-ans-role-update-kernel/.travis.yml

```diff
@@ -0,0 +1,25 @@
+services: docker
+
+env:
+- distro: centos7
+- distro: centos6
+- distro: fedora27
+- distro: ubuntu1710
+- distro: ubuntu1604
+- distro: ubuntu1404
+
+script:
+  # Configure test script so we can run extra tests after playbook is run.
+  - export container_id=$(date +%s)
+  - export cleanup=false
+
+  # Download test shim.
+  - wget -O ${PWD}/tests/test.sh https://gist.github.com/brentwg/64f90bdda32a51360f71558d3171fcbb/raw/
+  - chmod +x ${PWD}/tests/test.sh
+
+  # Run tests.
+  - ${PWD}/tests/test.sh
+
+notifications:
+  webhooks: https://galaxy.ansible.com/api/v1/notifications/
+
```

Comment:
I think this whole file could be deleted. We are not using .travis per role, but a single one at the root of the repo.

Reply:
done. I will submit the commit.